packages/flutter/test/widgets/clip_test.dart

```diff
@@ -431,7 +433,74 @@ void main() {
     );
   });
 
-  testWidgets('PhysicalModel painting', (WidgetTester tester) async {
+  // We should not rotate here if we want to test different bleeding edge
+  // artifacts. If rotated, the inner container will draw a rotated rect in
+  // anti-alias and that will cause some bleeding edges due to anti-aliasing
+  // regardless of what clip we have (i.e., not matter what clipBehavior we
+  // have, paths are always drawn in anti-alias).
+  const double physicalRotatianRadians = 0.0;
```

Comment:
i don't think this is accurate. If it is, then we need to find a solution. People don't want or expect bleeding in these cases.

Reply:
According to our discussion offline, change rotation back to 1.0 radians, and comment that there should only be bleeding edges on rect edges (but not round corners).